web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -140,6 +140,39 @@ export default class Client extends EventEmitterWebAuthnSender {
     };
   }
 
+  private async initWasm() {
+    // select the wasm log level
+    let wasmLogLevel = LogType.OFF;
+    if (import.meta.env.MODE === 'development') {
+      wasmLogLevel = LogType.TRACE;
+    }
+
+    await init();
+    init_wasm_log(wasmLogLevel);
+  }
+
+  private initFastPathProcessor(ioChannelId: number, userChannelId: number) {
+    if (!this.spec.width || !this.spec.height) {
+      this.handleError(
+        new Error(
+          'client screen spec must be set before initializing fast path processor'
```

Comment:
Is this user-facing? The message seems geared towards a dev (I would not expect a user to have any idea what to do here).

I would use `this.logger` for errors geared towards developers, and put more user-friendly messages in errors that are surfaced to the user

Reply:
It's basically an error that should never happen. Maybe I should add a request to contact teleport support?

Reply:
Or maybe just a more generic "internal error" message